### ISSUE TITLE
More reactions that just like/heart.

### ISSUE THREAD

**Issue by trusktr** [bluesky-social/social-app#6911]
### Describe the Feature

It would be great to have different types of reactions on posts so we can get a better idea of the sentiment people feel towards them. This would be useful for labelers, feeds, and other custom algos.

### Additional Context

When there is something that some people dislike, and some people like, it can be confusing, because you may have to "like" something you actually dislike in order to help make it more visible, and people who like it may be liking something labeled negatively.

Instead, it would be much better to be able to give negative reactions to things you don't like or oppose, and positive reactions to the ones you like.

Imagine a post like this:

"X casualties in Y city for Z people"

This is a hypothetical post that could be both very negative, and very positive.

However, because it may be a heated topic, the only way to make it be highly engaged is to "like" it.

- people who are in opposition may give a false positive just to spread awareness.
- people in support may give a true positive.

But if the topic is, for sale of example, talking about killing people in a Holocaust, then because of all the "likes", it becomes impossible to tell who actually likes it, and who is in opposition, or a variety of other sentiments.

Just like labels can be used to label anyone in order to give each other objective insight, sentiments are as important, for example, because by seeing who likes something evil, or who hates something good for society (just examples) then people can make better decisions on who to follow, who to block, who to add to lists, who to label, etc.

So, by having only the ability to "like" posts, we are in restricted in our ability to analyze a posts as effectively as we could. The ability to also consider sentiment from reactions would improve our ability to create better custom feed/etc algorithms.

**Comment by rosekamallove** [comment]
What reactions are we thinking of adding?

I am thinking in terms of what LinkedIn offers, it allows for a very good variation in the type of replies a person can give to a particular post (for sentiment analysis) although it doesn't allow for disliking a post.

**Comment by SpriterGors** [comment]
I'm kind of wary about putting more reactions, because depending on what they are, it can and will be weaponized (such as the perennial Facebook angry and haha reacts to dunk on others).

**Comment by akerbeltz** [comment]
I agree with the need for being careful about what is added, but at the very least, we could bring the Star back - what Twitter had in the old days, which was much superiour to the Heart because you could star something (like bad news) for attention but without implying you're loving the news about, say, the earthquake in Myanmar.

**Comment by cooljeanius** [comment]
I personally liked the way Miiverse did things, where the styling of the "Yeah!" button was tied to the face the original poster used... it helped promote empathy and discourage emotionally inappropriate reactions.
